src/includes/attachment-upload/android.mdx

```diff
@@ -0,0 +1,37 @@
+```java
```

Comment:
H: Also not here, we could just remove this file if its exactly the same as Java